py4DSTEM/preprocess/__init__.py

```diff
@@ -1,6 +1,5 @@
 from .preprocess import *
 from .darkreference import *
 from .electroncount import *
-from .relativity import *
```

Comment:
has this functionality been removed or moved elswhere?

Reply:
removed, as noted in the description of the PR